xml/abstract_commands.xml

```diff
@@ -91,37 +97,53 @@
             Table~\ref{tab:datareg}.
         </field>
         <field name="aarpostincrement" bits="19">
-            0: No effect. This variant must be supported.
+            <value v="0" name="disabled">
+            No effect. This variant must be supported.
+            </value>
 
-            1: After a successful register access, \FacAccessregisterRegno is
+            <value v="1" name="enabled">
+            After a successful register access, \FacAccessregisterRegno is
             incremented. Incrementing past the highest supported value
             causes \FacAccessregisterRegno to become \unspecified.  Supporting
             this variant is optional. It is undefined whether the increment
             happens when \FacAccessregisterTransfer is 0.
+            </value>
         </field>
         <field name="postexec" bits="18">
-            0: No effect. This variant must be supported, and is the only
+            <value v="0" name="disabled">
+            No effect. This variant must be supported, and is the only
             supported one if \FdmAbstractcsProgbufsize is 0.
+            </value>
 
-            1: Execute the program in the Program Buffer exactly once after
+            <value v="1" name="enabled">
+            Execute the program in the Program Buffer exactly once after
             performing the transfer, if any. Supporting this variant is
             optional.
+            </value>
         </field>
         <field name="transfer" bits = "17">
-            0: Don't do the operation specified by \FacAccessregisterWrite.
+            <value v="0" name="disabled">
+            Don't do the operation specified by \FacAccessregisterWrite.
+            </value>
 
-            1: Do the operation specified by \FacAccessregisterWrite.
+            <value v="1" name="enabled">
+            Do the operation specified by \FacAccessregisterWrite.
+            </value>
 
             This bit can be used to just execute the Program Buffer without
             having to worry about placing valid values into \FacAccessregisterAarsize or \FacAccessregisterRegno.
         </field>
         <field name="write" bits="16">
             When \FacAccessregisterTransfer is set:
-            0: Copy data from the specified register into {\tt arg0} portion
+            <value v="0" name="arg">
```

Comment:
Should the value names on 0 and 1 be read and write instead of arg and register?

Reply:
I chose this because the field is already called write. So now you're setting write=arg or write=register, which makes more sense to me than write=write or write=read.

If I were renaming the field as well, maybe direction=into_arg and direction=into_register?

Reply:
Oh, I see.  I have always thought of this as read/write of the hart register which does a write/read of data0 (plus maybe data1 or more) as a necessary part of that operation.  But it's pretty clear from the text here and elsewhere that reading a hart register will write "arg0" which is some number of data registers.  If you call it "arg0" instead of "arg" then I think it makes sense and is consistent with everything else.

Reply:
Done.